### ISSUE TITLE
Perl is broken

### ISSUE THREAD

**Issue by Neo-Oli** [termux/termux-packages#229]
Several things dependent on Perl don't work correctly anymore. For example the `mosh` wrapper script throws:
```
Can't load '/data/data/com.termux/files/usr/lib/perl5/5.22.1/arm-linux/auto/IO/IO.so' for module IO:
 dlopen failed: cannot locate symbol "PL_memory_wrap" referenced by "IO.so"... at /data/data/com.ter
mux/files/usr/lib/perl5/5.22.1/XSLoader.pm line 70.
 at /data/data/com.termux/files/usr/lib/perl5/5.22.1/arm-linux/IO.pm line 11.
Compilation failed in require at /data/data/com.termux/files/usr/lib/perl5/5.22.1/arm-linux/IO/Handl
e.pm line 269.
BEGIN failed--compilation aborted at /data/data/com.termux/files/usr/lib/perl5/5.22.1/arm-linux/IO/H
andle.pm line 269.
Compilation failed in require at /data/data/com.termux/files/usr/lib/perl5/5.22.1/arm-linux/IO/Socke
t.pm line 12.
BEGIN failed--compilation aborted at /data/data/com.termux/files/usr/lib/perl5/5.22.1/arm-linux/IO/Socket.pm line 12.
Compilation failed in require at /data/data/com.termux/files/usr/bin/mosh line 36.
BEGIN failed--compilation aborted at /data/data/com.termux/files/usr/bin/mosh line 36.
```
The following throw similar errors:
- cowsay
- parallel

This is possibly also the reason I can't install buffer.pl in #210

**Comment by fornwall** [comment]
Yes, building perl as a shared library, to enable embedding in other programs such as weechat, broke perl (it's the same reason causing buffer.pl to not work). 

Will look at it if it can be fixed shortly, else we'll have to revert building perl as a shared library for now.

**Comment by fornwall** [comment]
Thanks - this is now fixed in the `5.22.1-4` version of the perl package.
